### ISSUE TITLE
Not working when installed inside roots/bedrock environment

### ISSUE THREAD

**Issue by mortensassi** [palmiak/timber-acf-wp-blocks#59]
Thanks for your great extension! Unfortunately when i install the plugin in the root of my bedrock setup, my blocks won't get recognized in the backend. However, installing this plugin inside the theme-directory works. I guess it has something to do with timing and if the Timber-Class exists?

**Comment by palmiak** [comment]
Installing it inside of a theme is the recommended way.

However maybe it could be possible to make it working while installing in
the root composer file.

czw., 25 lis 2021, 18:02 użytkownik Moe ***@***.***> napisał:

**Comment by mortensassi** [comment]
Yeah you can see bedrock loads first the dependencies but i guess in that moment nor ACF or Timber get recognized. When i think about deployment later, i guess its more suitable when theme-related extensions are part of the theme.

**Comment by merijnponzo** [comment]
In my bedrock setup it's working with both installation methods (plugin, or directly from autoload)

In my composer
```
"palmiak/timber-acf-wp-blocks": "^1.15"
```
don't forget to 
```
composer dump-autoload
```
after your install

and in my theme
```
// init timber
$timber = new \Timber\Timber();
// init timber acf blocks
if ( class_exists( 'Timber_Acf_Wp_Blocks' ) ) {
	new Timber_Acf_Wp_Blocks();
   
}
```
